Sofa/Component/Mass/src/sofa/component/mass/UniformMass.h

```diff
@@ -78,6 +77,13 @@ class UniformMass : public core::behavior::Mass<DataTypes>
     DataSetIndex     d_indices; ///< optional local DOF indices. Any computation involving only indices outside of this list are discarded
     Data<bool> d_preserveTotalMass; ///< Prevent totalMass from decreasing when removing particles.
 
+    /// Enumeration specifying which data was used for initialization
+    enum InitMethod {
+        totalMass,
+        vertexMass
+    };
+    InitMethod m_initMethod = totalMass;
+
```

Comment:
I am surprised that there are no guidelines for enum types in the doc. But generally:

- the values should be UpperCamelCase or UPPERCASE
- `enum class` is recommended: https://stackoverflow.com/questions/18335861/why-is-enum-class-considered-safer-to-use-than-plain-enum

Reply:
Seems the UPPERCASE is the most used case in the codebase, indeed.
But I dont see why you put a `= 0`. (and add a new line for the open curling brace 🫢)